ttnn/ttnn/operations/conv2d.py

```diff
@@ -104,8 +176,16 @@ def conv2d(
     memory_config: ttnn.MemoryConfig = None,  # memory config overrides by user
     conv_op_cache={},  # basic conv object caching in python needed for intermediate refactoring. Not needed after full op refactoring in C++.
     debug=False,  # ignored
+    return_output_size=False,
+    return_prepared_device_weights=False,
```

Comment:
Why complicate the API like this?

If the values are computed then there's no additional cost to returning them.

Reply:
I was under the impression we wanted the api to return the output alone by default

Reply:
@mywoodstock what are your thoughts?

Reply:
Yes, thats the purpose of this change to return only the output as pytorch.